src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -287,6 +287,11 @@ namespace winrt::TerminalApp::implementation
         ShowSetAsDefaultInfoBar();
     }
 
+    Windows::UI::Xaml::Automation::Peers::AutomationPeer TerminalPage::OnCreateAutomationPeer()
+    {
+        return Automation::Peers::FrameworkElementAutomationPeer(*this);
```

Comment:
is one not automatically created for us?

Reply:
aka why is this necessary? Is it because a page does not have an automation peer already?

Reply:
Surprisingly, `Automation::Peers::FrameworkElementAutomationPeer::FromElement(*this))` returns `nullptr` if we don't have `OnCreateAutomationPeer()` defined. So all of the `if (auto autoPeer = FromElement(*this))` don't get executed. Why? Not sure. But in practice, I know that's the case haha.